From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Wed, 29 Jul 2026 22:41:44 +0700
Subject: [PATCH] fixup! pubspec: drop riverpod_generator

last usage is in last.fm scrobber, which is gone

Signed-off-by: fukiame <fukiame@proton.me>

diff --git a/pubspec.yaml b/pubspec.yaml
index 0aca5dd5..d3e65159 100644
--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -80,7 +80,6 @@ dev_dependencies:
     sdk: flutter
   isar_community_generator: ^3.3.2
   json_serializable: 6.11.2
-  riverpod_generator: 4.0.0+1
 
 flutter_launcher_icons:
   image_path: "assets/icons/reverse_flick_logo.png"
-- 
2.55.0

